packages/terra-dialog/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+## 2.65.0 - (January 9, 2023)
+
+* Changed
+  * Minor dependency version bump
```

Comment:
```suggestion
  * Minor dependency version bump of terra-icon
```
So that we can track these changes with further more information. right now there are quite a few logs with same entry "Minor dependency version bump" it would be difficult to track the changes that got released with specified version.

Reply:
Updated [461687c](https://github.com/cerner/terra-core/pull/3711/commits/461687c1c2795b60aea8ab3d0350f0de7ecbc6cd)